### PR TITLE
[Fix](executor)Fix query has no coord add may cause report failed.

### PR DESCRIPTION
**Pull request by wangbo**
## Proposed changes

1 Add a column query_type for schema table ```backend_active_tasks```.
2 For external query, they has no coord addr, should skip it when report audit log to Fe.
3 Add strictly check for coord addr and QueryType in debug mode.

### CODE REVIEW

**Comment by wangbo** [comment]
run buildall

File: be/src/runtime/runtime_query_statistics_mgr.h [hunk 1]
```diff
@@ -57,6 +58,7 @@ class QueryStatisticsCtx {
     std::vector<std::shared_ptr<QueryStatistics>> _qs_list;
     bool _is_query_finished;
     TNetworkAddress _fe_addr;
+    TQueryType::type _query_type;
```
Comment (yiguolei) [inline be/src/runtime/runtime_query_statistics_mgr.h:61]: It should be const member

**Comment by wangbo** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]

**Review by mrhhsg — APPROVED** [review]
LGTM

**Pull request merged**